backend/src/test/java/com/now/naaga/temporaryplace/presentation/TemporaryPlaceControllerTest.java

```diff
@@ -0,0 +1,157 @@
+                .multiPart(
+                        new MultiPartSpecBuilder(temporaryPlace.getPosition().getLatitude().setScale(6, RoundingMode.HALF_DOWN).doubleValue()).controlName(
+                                "latitude").charset(StandardCharsets.UTF_8).build())
+                .multiPart(
+                        new MultiPartSpecBuilder(temporaryPlace.getPosition().getLongitude().setScale(6, RoundingMode.HALF_DOWN).doubleValue()).controlName(
+                                "longitude").charset(StandardCharsets.UTF_8).build())
+                .multiPart(new MultiPartSpecBuilder(new FileInputStream(new File("src/test/java/com/now/naaga/temporaryplace/fixture/루터회관.png"))).controlName(
+                                "imageFile").charset(StandardCharsets.UTF_8)
+                        .fileName(
+                                "src/test/java/com/now/naaga/temporaryplace/fixture/루터회관.png")
+                        .mimeType(
+                                "image/png")
+                        .build())
+                .header("Authorization", "Bearer " + accessToken)
+                .when()
+                .post("/temporary-places")
+                .then()
+                .log().all()
+                .extract();
+        final int statusCode = extract.statusCode();
+        final String location = extract.header("Location");
+        final TemporaryPlaceResponse actual = extract.as(TemporaryPlaceResponse.class);
+        final TemporaryPlaceResponse expected = TemporaryPlaceResponse.from(temporaryPlace);
+
+        //then
+        assertSoftly(softAssertions -> {
+            softAssertions.assertThat(statusCode).isEqualTo(HttpStatus.CREATED.value());
+            softAssertions.assertThat(location).isEqualTo("/temporary-places/" + actual.id());
+            softAssertions.assertThat(actual)
+                    .usingRecursiveComparison()
+                    .ignoringFields("id", "imageUrl")
+                    .isEqualTo(expected);
+        });
+    }
+
+    @Test
+    void 검수할_장소_목록_조회_요청을_받으면_200_상태코드와_함께_검수할_장소_목록을_응답한다() throws FileNotFoundException, JsonProcessingException {
+        //given
+        final TemporaryPlace temporaryPlace = temporaryPlaceBuilder.init()
+                .build();
+
+        //when
+        final ExtractableResponse<Response> extract = given()
+                .log().all()
+                .auth().preemptive().basic(id, password)
+                .when()
+                .get("/temporary-places")
+                .then()
+                .log().all()
+                .extract();
+        final int statusCode = extract.statusCode();
+        final String jsonResponse = extract.body().asString();
+        final List<TemporaryPlaceResponse> actual = objectMapper.readValue(jsonResponse, new TypeReference<List<TemporaryPlaceResponse>>() {
+        });
```

Comment:
P2:

```suggestion
final List<TemporaryPlaceResponse> actual = extract.as(new TypeRef<>(){});
```

extract 객체에서 List 형식으로 결과 값을 직렬화시킬때는 위 코드처럼 작성해주셔도 됩니다 ㅎㅎ

Reply:
공부해오겠습니다... ㅜㅜ